test/testScan.py

```diff
@@ -32,6 +31,7 @@ def testnumindices(self):
         else:
             self.assertEquals(len(self.data.optdone), 12 + self.extra)
 
+    @skipForParser('ORCA', "?")

```

Comment:
I don't know why this test was skipped -- you added the exception in 2014 according to blame.

I think many of the exceptions currently applied can now be lifted. But I didn't explore that direction for this PR.